### PR TITLE
[New Package] tar

### PR DESCRIPTION
**Pull request by saltedcoffii**
Add new package GNU tar. Although tar exists in chromium os, this should provide an updated binary, header files and manpages.

Compiles fine on x86_64 and should compile fine on arm but hasn't been tested.

### CODE REVIEW

File: packages/tar.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Tar < Package
+  description 'GNU Tar provides the ability to create tar archives, as well as various other kinds of manipulation.'
+  homepage 'https://www.gnu.org/software/tar/'
+  version '1.33'
+  compatibility 'all'
+  source_url 'https://ftp.gnu.org/gnu/tar/tar-1.33.tar.xz'
```
Comment (uberhacker) [inline packages/tar.rb:8]: Better source? https://ftpmirror.gnu.org/tar/tar-1.33.tar.xz

**Comment by uberhacker** [comment]
Thank you for the new package @saltedcoffii!  You are on a roll.

**Comment by saltedcoffii** [comment]
Source changed ✅

**Comment by satmandu** [comment]
Does this have a xzutils dependency, or is that in base?

**Pull request merged**